### DISCUSSION TITLE
Windwos Server 2012 Support

### DISCUSSION THREAD

**Discussion by ggoyal23** [vcsjones/AzureSignTool#210] [General]
Hi,
I know that Azure Signtool is currently supported by Windows 2016 server or later but unfortunately we are still using Windows server 2012 version. Is it possible somehow (or any workaround) so it can supported by this server version.

Regards,
Gaurav

**Comment by VaronisContributor** [comment]
Try taking the mssign32.dll file from Windows 2016 and put it into the working folder of the tool. If it doesn't crash instantly, it might actually work.

**Comment by awnishupadhyay** [comment]
Hi Gaurav, I am in the same boat and couln't get the Azure sign tool work with Windows server 2012. I tried the above step of copying mssign32.dll but with no luck. If you managed to resolve the issue, could you share any work around here.

Thanks
Awnish

**Comment by ggoyal23** [comment]
Hi @awnishupadhyay  No unfortunately couldn't find the solution so we created new latest windows  server and signing the files from there.

**Comment by yuvnith** [comment]
Greetings, has anyone succeeded in getting AzureSignTool to function on Windows Server 2012?
